### PR TITLE
1flow-web actions

### PR DESCRIPTION
**Pull request by namit1Flow**
1Flow web actions doc,we are targeting on track and identify user.

### CODE REVIEW

File: src/connections/destinations/catalog/1Flow Web Actions/index.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
+---
+title: 1Flow Web Actions
+rewrite: true
+id: 62bf80378e3d0241ab190594
```
Comment (forstisabella) [inline src/connections/destinations/catalog/1Flow Web Actions/index.md:4]: hi @namit1Flow - looks like this ID corresponds to the 1Flow Analytics destination. Are you looking to create docs for that destination?

File: src/connections/destinations/catalog/1Flow Web Actions/index.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+## Components
+
+- **Browser**
+
+---
+## Connection Modes
+ ## Device-mode
+ - Web
+
+---
```
Comment (forstisabella) [inline src/connections/destinations/catalog/1Flow Web Actions/index.md:22]: This information is autogenerated when the docs build!
```suggestion

```

File: src/connections/destinations/catalog/1Flow Web Actions/index.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+
+Knowing who your users are and what they're doing unlocks more advanced filtering and targeting capabilities across all of 1Flow's tools, helping you find meaningful insights faster.
+
+This destination is maintained by 1Flow. For any issues with the destination, [contact the 1Flow Support team](mailto:support@1flow.app).
+
+## Getting Started
+
+
+
+1. Navigate to **Connections** and click **Add Destination** From the Segment web app.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/1Flow Web Actions/index.md:40]:
```suggestion
1. From the Segment web app, navigate to **Connections** and click **Add Destination** .
```

**Review by forstisabella — CHANGES_REQUESTED** [review]

File: src/connections/destinations/catalog/1Flow Web Actions/index.md (hunk 1 above)
Comment (tcgilbert) [reply src/connections/destinations/catalog/1Flow Web Actions/index.md:4]: this will be the correct ID: `656773f0bd79a3676ab2733d`

Comment (tcgilbert) [inline src/connections/destinations/catalog/1Flow Web Actions/index.md:4]:
```suggestion
id: 656773f0bd79a3676ab2733d
```

this will be the correct ID ^

**Review by forstisabella — APPROVED** [review]
a few changes and we'll be good to go 🚀

File: src/connections/destinations/catalog/actions-1flow/index.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
+---
+title: 1Flow Web Actions
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-1flow/index.md:2]:
```suggestion
title: 1Flow Web (Actions)
```

**Comment by namit1Flow** [comment]
Hi @forstisabella , @tcgilbert ,

Thank you for reviewing; I have made the suggested change.

**Comment by tcgilbert** [comment]
hi @namit1Flow and @forstisabella - apologies for missing this, but it appears that you did in fact submit docs for this destination previously. 

[I have created a new PR](https://github.com/segmentio/segment-docs/pull/5774) that corrects for this so I will close this one out.

**Pull request closed without merging**